### ISSUE TITLE
Can we have a sorting on the basis of the newest Event first on the list

### ISSUE THREAD

**Issue by vikramtanwar** [coloredcow/WP-RSVP#74]
![image](https://user-images.githubusercontent.com/24538347/30577582-d62e5980-9d2c-11e7-829a-180e76cf876c.png)

**Comment by wevekamola** [comment]
@vikramtanwar so right now sorting is happing according to the date of the event reason for that is It will be easy for the organizer to set the priorities for the event by seeing them sorted in date wise format of the event.

**Comment by abhi1203** [comment]
@vikramtanwar i agree, let's have it sorted as per -> most recent event first because that will be most important for the creator. 
But since vikram's point is also valid, we can have sorting there newest event and most recent event. But we can target this later, what do you say vikram?

**Comment by wevekamola** [comment]
Yes, Abhishek I like the idea of providing an option for sorting...newest event and recent event.
